webpack.config.js

```diff
@@ -0,0 +1,24 @@
+var path = require('path')
+var webpack = require('webpack')
+
+module.exports = {
+  entry: "./static/components/PoiCards.js",
+  output: {
+    path: "./static/components",
+    filename: "App-around.js"
```

Comment:
Considering [the React naming convention](https://gist.github.com/koistya/d7a507438c741ee6adb5), it should be called `AppAround.js`